mnefun/_mnefun.py

```diff
@@ -1458,7 +1576,7 @@ def _raw_LRFCP(raw_names, sfreq, l_freq, h_freq, n_jobs, n_jobs_resample,
         print('    Loading and filtering %d files.' % len(raw_names))
     raw = list()
     for rn in raw_names:
-        r = Raw(rn, preload=True)
+        r = Raw(rn, preload=True, allow_maxshield=allow_maxshield)
```

Comment:
you need to wrap this in `with warnings.catch_warnings(record=True):` to catch the warning